templates/scripts/add-vpn-user.sh

```diff
@@ -65,11 +85,12 @@ cat <<-EOF > ${CONFIG_FILE}
 PrivateKey = $client_private_key
 ListenPort = 34567
 Address = $next_ip/32
+DNS = $dns_server
```

Comment:
Isn't this just setting your dns to what was already set in your resolv.conf?

Reply:
Well, WireGuard client will handle it per OS. For MacOS by default, it doesn't change existing /etc/resolv.conf, but deal with mDNSResponder and the result can be found by running `scutil --dns`

Reply:
Okay, didn't see that you were using the `$EXEC` var earlier to pull the dns from the pod.

Reply:
as discussed, that $dns_server is retrieved from vpn pod (not local)